### PR TITLE
Add GHA

### PR DESCRIPTION
**Pull request by ronaldtse**

### CODE REVIEW

**Comment by ronaldtse** [comment]
@CAMOBAP can you help update this PR so that it builds the Ribose flavor (non-default gem)? Thanks!

File: .github/workflows/pages.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+      - uses: actions/cache@v2
+        with:
+          path: ~/.metanorma-ietf-workgroup-cache.json
+          key: metanorma-ietf-workgroup-cache
+          restore-keys: metanorma-ietf-workgroup-cache
+
+      - uses: metanorma/metanorma-build-scripts/gh-rubygems-setup-action@master
+        with:
+          token: ${{ secrets.METANORMA_CI_PAT_TOKEN }}
```
Comment (CAMOBAP) [inline .github/workflows/pages.yml]:
```suggestion
      - run: |
          curl -L --retry 3 https://raw.githubusercontent.com/metanorma/metanorma-build-scripts/master/gemfile-to-bundle-add.sh | bash
```

**Comment by CAMOBAP** [comment]
@ronaldtse do we really need sassc here?

**Comment by ronaldtse** [comment]
Technically no need for sassc. But are we facing the need for sassc here?

**Comment by ronaldtse** [comment]
Ah, we should not use the git gem.

**Comment by ronaldtse** [comment]
Thank you @CAMOBAP !

**Pull request merged**